modules/ROOT/pages/design-create-publish-api-fragment.adoc

```diff
@@ -37,7 +42,11 @@ _Result:_ The text editor opens. The editor is divided into three panels:
 +
 * The left panel lists the files in your project.
 +
-By default, the editor creates a RAML file that has the same name as the project. This file is set as the root file of the project.
+You can create new files by clicking *+*, and then *New file*. Select the API format of the file between RAML 1.0, OAS 3.0, JSON schema, or other. Select the file type if needed, name your file, and click *Create*.
++
+By default, the editor creates a file that has the same name as the project. This file is set as the root file of the project.
++
+You can set YAML, RAML, or JSON files as your root file.
```

Comment:
I would replace "JSON" with "JSON Schema" since we cannot set any JSON as root file. It needs to be a json schema file